### PR TITLE
New Tenable.ad data connector and parser

### PR DESCRIPTION
**Pull request by fr4nco1s**
## Proposed Changes

  - Tenable.ad data connector
  - Tenable.ad kusto parser

### CODE REVIEW

**Comment by v-jayakal** [comment]
@fr4nco1s - Can you please consider this as making it a solution instead of data connector? You can refer existing solutions here https://github.com/Azure/Azure-Sentinel/tree/master/Solutions for reference. Thanks.

**Comment by fr4nco1s** [comment]
> @fr4nco1s - Can you please consider this as making it a solution instead of data connector? You can refer existing solutions here https://github.com/Azure/Azure-Sentinel/tree/master/Solutions for reference. Thanks.

Hi @v-jayakal I just created a TenableAD solution and moved the connector data and parser inside.
Let me know if I need to change something.
Thank you for your time.

File: Logos/tenablead_logo.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg id="5eb4e179-601a-4ee2-b232-fd4ded4f84cd" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 326 72.5">
```
Comment (v-sabiraj) [inline Logos/tenablead_logo.svg:1]: Logo size should be less than 5kb. Please update.

**Review by v-sabiraj — CHANGES_REQUESTED** [review]

File: Logos/tenablead_logo.svg (hunk 1 above)
Comment (fr4nco1s) [reply Logos/tenablead_logo.svg:1]: Hi @v-sabiraj , I asked our graphic design team to make the logo less than 5kb, but it seems that's it's too much tiny..
Any possiblity that you make an exception and accept this logo please ?

Comment (v-sabiraj) [reply Logos/tenablead_logo.svg:1]: Hi @fr4nco1s , We can't accept this logo, it is mandatory to have logo size less than 5kb. Please update it.

Comment (fr4nco1s) [reply Logos/tenablead_logo.svg:1]: Hi @v-sabiraj , we managed to minify the svg logo size under 5kb.

**Comment by NikTripathi** [comment]
@oshezaf Please review parser. Thanks.

**Review by NikTripathi — APPROVED** [review]
Data connector looks good. Can merge once parser is reviewed.

**Comment by oshezaf** [comment]
@NikTripathi : as mentioned in the past, I am not the person to review parsers. It is not a trivial task that requires knowing the specific log sources well. I will be reviewing ASIM compliant parsers for ASIM compliancy, but not the parsing logic, once we start creating ASIM compliant parsers.

Tagging @ashishsyal

**Comment by v-sabiraj** [comment]
Hey @fr4nco1s, We have resolved the issue for PR Validations failure and request you to please update your PR branch with the latest changes from **Azure:master** so that the PR Validations can be executed.... Thanks!!!

**Comment by fr4nco1s** [comment]
> Hey @fr4nco1s, We have resolved the issue for PR Validations failure and request you to please update your PR branch with the latest changes from **Azure:master** so that the PR Validations can be executed.... Thanks!!!

Hi @v-sabiraj I merged with Azure:master and got the PR Validations can be executed.

**Pull request merged**